generated/google_cloud_longrunning/lib/longrunning.dart

```diff
@@ -201,11 +203,24 @@ final class ListOperationsRequest extends ProtoMessage {
   /// The standard list page token.
   final String? pageToken;
 
+  /// When set to `true`, operations that are reachable are returned as normal,
```

Comment:
[nit] Consider rephrasing to start with "Whether".  https://dart.dev/effective-dart/documentation#prefer-starting-a-boolean-variable-or-property-comment-with-whether-followed-by-a-noun-or-gerund-phrase